Components/components.css

```diff
@@ -1 +1,2 @@
-/* Import all component css files here */
\ No newline at end of file
+/* Import all component css files here */
+@import url(/Components/Alerts/alerts.css);
```

Comment:
The file and folder names are in the Upper camel case it should be in kebab case example "alert-presentation".

Reply:
Thank you I didn't know this.
Done!